src/main/java/tetris/Grid.java

```diff
@@ -83,11 +83,11 @@ int clearLines() {
             if (skipRow) {
                 row++;
             } else {
-                score += localBoard.length;
+                rowsCleared += 1;
```

Comment:
Does exactly the same ;)  += is a bit more Pythonic, my language of choice. ++ is more Javanian I guess.